src/appfl/algorithm/scheduler/base_scheduler.py

```diff
@@ -0,0 +1,62 @@
+import abc
+from omegaconf import DictConfig
+from concurrent.futures import Future
+from typing import Union, Dict, Any, Tuple, OrderedDict
+
+class BaseScheduler:
+    def __init__(
+        self, 
+        scheduler_configs: DictConfig, 
+        aggregator: Any,
+        logger: Any
```

Comment:
Can we specify types of these attributes?

Reply:
Yes, I changed to `BaseAggregator`